x/supplier/types/message_create_claim_test.go

```diff
@@ -6,37 +6,104 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/pokt-network/poktroll/testutil/sample"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
-// TODO(@bryanchriswhite): Add unit tests for message validation when adding the business logic.
-
 func TestMsgCreateClaim_ValidateBasic(t *testing.T) {
 	tests := []struct {
-		name string
-		msg  MsgCreateClaim
-		err  error
+		testCase string
```

Comment:
I know I asked already in another PR but I'm commenting here to track this instance as well. Why not `name`?

Reply:
Will stick to `desc`